docs/src/content/docs/fr/guides/customization.mdx

```diff
@@ -0,0 +1,436 @@
+
+<Tabs>
+  <TabItem label="Frontmatter">
+
+```md
+---
+# src/content/docs/example.md
+title: Page avec seulement les H2s dans la table des matières
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 2
+---
+```
+
+  </TabItem>
+  <TabItem label="Global config">
+
+```js
+// astro.config.mjs
+
+defineConfig({
+  integrations: [
+    starlight({
+      title: "",
+      tableOfContents: { minHeadingLevel: 2, maxHeadingLevel: 2 },
+    }),
+  ],
+});
+```
+
+  </TabItem>
+</Tabs>
+
+Désactivez la table des matières complètement en définissant l’option `tableOfContents` à `false`:
+
+<Tabs>
+  <TabItem label="Frontmatter">
+
+```md
+---
+# src/content/docs/example.md
+title: Page sans table des matières
+tableOfContents: false
+---
+```
+
+  </TabItem>
+  <TabItem label="Global config">
+
+```js
+// astro.config.mjs
+
+defineConfig({
+  integrations: [
+    starlight({
+      title: "Site avec la table des matières désactivée globalement",
+      tableOfContents: false,
+    }),
+  ],
+});
+```
+
+  </TabItem>
+</Tabs>
+
+## Liens sociaux
+
+Starlight supporte par défaut l’ajout de liens vers vos comptes de médias sociaux dans l’en-tête du site via l’option [`social`](/reference/configuration/#social) dans l’intégration Starlight.
+
+Actuellement, les liens vers Discord, GitHub, Mastodon et Twitter sont pris en charge.
+Faites-nous savoir sur GitHub ou Discord si vous avez besoin de la prise en charge d’un autre service !
+
+```js
+// astro.config.mjs
+import { defineConfig } from "astro/config";
+import starlight from "@astrojs/starlight";
+
+export default defineConfig({
+  integrations: [
+    starlight({
+      title: "Site avec des liens sociaux",
+      social: {
+        discord: "https://astro.build/chat",
+        github: "https://github.com/withastro/starlight",
+        mastodon: "https://m.webtoo.ls/@astro",
+        twitter: "https://twitter.com/astrodotbuild",
+      },
+    }),
+  ],
+});
+```
+
+## Liens d’édition de page
+
+Starlight peut afficher un lien "Modifier la page" dans le pied de page de chaque page.
+Cela permet au lecteur de trouver facilement le fichier à modifier pour améliorer vos documents.
+Pour les projets open source en particulier, cela peut aider à encourager les contributions de votre communauté.
+
+Pour activer les liens de modification, définissez l’option de configuration [`editLink.baseUrl`](/reference/configuration/#editlink) en lui assignat l’URL utilisée pour modifier votre référentiel. Cela se passe dans la configuration de l’intégration Starlight.
+La valeur de `editLink.baseUrl` sera automatiquement ajoutée au chemin d’accès vers la page actuelle pour former le lien d’édition complet.
+
+Les motifs courants incluent:
```

Comment:
I was looking for a way to translate "Patterns", and motif was the best choice for me. But maybe you are right and instead of trying a direct translation, maybe think differently.
What do you thing about : "Les formes d’URL les plus courantes incluent :"

Reply:
Great suggestion, I think it conveys the meaning perfectly.

Reply:
OK, I did include this version.